torch/_higher_order_ops/wrap.py

```diff
@@ -82,9 +82,10 @@ class TagActivationCheckpoint(HigherOrderOperator):
     in the forward and recomputed forward in backward.
     """
 
+    context_fn_list = []
```

Comment:
If we're going to be in the business of doing this, this should probably be TLS too